pkg/console/operator/operator.go

```diff
@@ -200,21 +208,33 @@ func (c *consoleOperator) deleteAllResources(cr *consolev1.Console) error {
 	return utilerrors.FilterOut(utilerrors.NewAggregate(errs), errors.IsNotFound)
 }
 
-// this may need to eventually live under each sync version, depending on if there is
-// custom sync logic
-func (c *consoleOperator) defaultConsole() *consolev1.Console {
-	logrus.Info("creating console CR with default values")
-	return &consolev1.Console{
+func (c *consoleOperator) defaultConsoleConfig() *configv1.Console {
+	return &configv1.Console{
+		ObjectMeta: metav1.ObjectMeta{
+			Name: api.ResourceName,
+		},
+		Spec: configv1.ConsoleSpec{
+			//Authentication: consoleconfigv1.ConsoleAuthentication{
+			//	LogoutRedirect: "",
+			//},
+		},
+	}
+}
+
+func (c *consoleOperator) defaultConsoleOperatorConfig() *operatorsv1.Console {
+	return &operatorsv1.Console{
 		ObjectMeta: metav1.ObjectMeta{
-			Name:      api.ResourceName,
-			Namespace: api.OpenShiftConsoleNamespace,
+			Name: api.ResourceName,
 		},
-		Spec: consolev1.ConsoleSpec{
+		Spec: operatorsv1.ConsoleSpec{
 			OperatorSpec: operatorsv1.OperatorSpec{
-				// by default the console is managed
 				ManagementState: operatorsv1.Managed,
+				// LogLevel:        "",
 			},
-			Count: consoleReplicas,
+			// Customization: consoleoperatorv1.ConsoleCustomization{
```

Comment:
Default should not populate these values, but they should be honored if set.

Reply:
Remove commented out code